src/ms/deconvolution.js

```diff
@@ -0,0 +1,43 @@
+import { ngmca } from 'ml-ngmca';
+
+/**
+ * Performing non-negative matrix factorization solving argmin_(A >= 0, S >= 0) 1 / 2 * ||Y - AS||_2^2 + lambda * ||S||_1
+ * @param {Chromatogram} chromatogram - GC/MS chromatogram where make the estimation.
+ * @param {Object} range - Range with from to and optional the sub-matrix with the mass x axis, times, and the number of pure components
```

Comment:
The shape of this object should be documented

Reply:
it's done